network.go

```diff
@@ -19,8 +19,17 @@ func networkStateChanged() {
 	// do not block the main thread
 	go waitCtrlAndRequestDisplayUpdate(true)
 
+	if timeSync != nil {
```

Comment:
These `nil` checks are necessary because we can hit `networkStateChanged` before we've even finished setting up.